h2o-core/src/main/java/hex/DistributionFactory.java

```diff
@@ -439,30 +421,116 @@ public QuantileDistribution(Model.Parameters params){
         super(params);
     }
 
+    @Override
+    public double deviance(double w, double y, double f) { return y > f ? w * quantileAlpha * (y - f) : w * (1 - quantileAlpha) * (f - y); }
+
+    @Override
+    public double negHalfGradient(double y, double f) { return y > f ? 0.5 * quantileAlpha : 0.5 * (quantileAlpha - 1); }
+}
+
+/**
+ * Custom distribution class to customized loss and prediction calculation.
+ * Currently supported only for GBM algorithm.
+ */
+class CustomDistribution extends Distribution {
+    
+    private CustomDistributionWrapper customDistribution;
```

Comment:
breaks h2o naming conventions - fields should start with `_`

this applies to the whole PR - please keep it consistent with the existing code base

Reply:
Ok, I will change it.